include/internal/hwcaps.h

```diff
@@ -176,6 +176,14 @@ CPU_FEATURES_START_CPP_NAMESPACE
 #define PPC_FEATURE2_HTM_NO_SUSPEND 0x00080000
 #endif
 
+#define RISCV_HWCAP_A (1UL << ('A' - 'A'))
```

Comment:
could you provider link to linux source for consistency?https://elixir.bootlin.com/linux/latest/source/arch/riscv/include/uapi/asm/hwcap.h

Reply:
what about `RISCV_HWCAP_I`, `RISCV_HWCAP_A`?

Reply:
Yes, I'll provide a link (I was intending to do that, but forgot) and include i and e.

Reply:
@michael-roe can you add the link please?

Reply:
Ok, done.